llorens/day04/Day04Solver.py

```diff
@@ -0,0 +1,42 @@
+from collections import defaultdict
+
+
+def solve_part_1(input):
+
+    lines = input.splitlines()
```

Comment:
File management and input preparation should be separated into another method IMO